test/sparsedir/cholmod.jl

```diff
@@ -185,10 +190,6 @@ ACSC = sprandn(10, 10, 0.3) + I
 @test ishermitian(Sparse(Hermitian(complex(ACSC), :L)))
 @test ishermitian(Sparse(Hermitian(complex(ACSC), :U)))
 
-# test Sparse constructor for c_Sparse{Tv,Ti} input( and Sparse*Sparse)
-B = CHOLMOD.Sparse(SparseMatrixCSC{Float64,Int32}(sprandn(48, 48, 0.1))) # A has Int32 indices
```

Comment:
`CHOLMOD.Sparse` multiplication is tested here 
https://github.com/JuliaLang/julia/blob/e432909e619174ad1ccb2dc778bb8d4fea12d7f8/test/sparsedir/cholmod.jl#L141-L169